common/src/main/java/com/caldevsupplychain/common/jwt/service/JwtService.java

```diff
@@ -0,0 +1,17 @@
+package com.caldevsupplychain.common.jwt.service;
+
+import com.caldevsupplychain.common.jwt.token.JWTAuthenticationToken;
+import com.caldevsupplychain.common.jwt.vo.JwtBean;
+import org.springframework.http.HttpHeaders;
+
+public interface JwtService {
+
+	JWTAuthenticationToken createJwtToken(JwtBean jwtBean);
```

Comment:
You can use `UserBean` instead of creating new Bean since the field is directly from user. Syntax wide is clearer to read `createJwtToken(UserBean user)` like create jwt token for user.

Reply:
fied